Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -164,6 +167,28 @@ class SearchViewController: SiteTableViewController,
         }
     }
 
+    private func loadFirefoxSuggestions() {
+        guard featureFlags.isFeatureEnabled(.firefoxSuggestFeature, checking: .userOnly) else {
+            return
+        }
+
+        profile.firefoxSuggest.interruptReader()
+
+        let tempSearchQuery = searchQuery
+        Task { [weak self] in
+            guard let suggestions = try? await self?.profile.firefoxSuggest.query(tempSearchQuery, types: [.sponsored, .nonSponsored]) else {
+                return
+            }
```

Comment:
nit: we try to keep our lines to 120 chars or less (though the swiftlint rule hasn't been implemented because there's still too many violations to fix before we can turn it on. Here, we can break on `query(` to fix that.

Reply:
I was wondering why `swiftlint` wasn't complaining, thanks for the context! Fixed here and elsewhere 😁